### PR TITLE
fix: Ensure wwwroot assets are copied properly

### PR DESCRIPTION
**Pull request by jeromelaban**
## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the new behavior?

Adjust for net9 publishing of wwwroot assets.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.4

File: src/Uno.Sdk/targets/Uno.Common.Wasm.targets [hunk 1]
```diff
@@ -83,13 +83,33 @@
...
+			<Content Remove="@(_wwwRootContentToUpdate)" />
+
+			<Content Include="@(_wwwRootContentToUpdate)">
+				<Link>%(RecursiveDir)%(Filename)%(Extension)</Link>
+				<UnoDeploy>Root</UnoDeploy>
+				<!-- net9+ compatibility -->
+				<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+			</Content>
+
+			<_wwwRootContentToUpdate Remove="@(_wwwRootContentToUpdate)" />
```
Comment (Youssef1313) [inline src/Uno.Sdk/targets/Uno.Common.Wasm.targets:112]: Originally, do we have every single file in `wwwroot` included in `Content`?

If `wwwroot` originally had `FileA` and `FileB`, but only `FileA` was included in `Content`. With these changes, we'll be adding both `FileA` and `FileB` as Content. Not sure if it's intended.

Comment (jeromelaban) [reply src/Uno.Sdk/targets/Uno.Common.Wasm.targets:112]: `wwwroot` is implicitly including everything, the same way it is with aspnet core apps. It got broken when the task above was introduced and we missed regression tests for that.

Comment (Youssef1313) [reply src/Uno.Sdk/targets/Uno.Common.Wasm.targets:112]: My comment means the other way around, does `Content` include everything in `wwwroot`? (not that wwwroot contains everything from Content)

Comment (Youssef1313) [reply src/Uno.Sdk/targets/Uno.Common.Wasm.targets:112]: Also, how things are going with non-Uno.Sdk?

Comment (jeromelaban) [reply src/Uno.Sdk/targets/Uno.Common.Wasm.targets:112]: Duplicates in this context are handled properly by the bootstrapper. Without the uno.sdk is different, it's managed by the bootstrapper itself.

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.4

**Pull request merged**